semantic_release/history/__init__.py

```diff
@@ -25,6 +25,14 @@
 logger = logging.getLogger(__name__)
 
 
+prerelease_pattern = f"-{config.get('prerelease_tag')}.\d+"
+version_pattern = f"(\d+.\d+.\d+({prerelease_pattern})?)"
+release_version_pattern = f"(\d+.\d+.\d+(?!{prerelease_pattern}))"
+
+release_version_regex = rf"{release_version_pattern}"
+version_regex = rf"{version_pattern}"
```

Comment:
I added this variables as single source of truth for regex version matching, for ease of use once as string and once as regex.